dev/package-examples/nginx-1.2.0/dataset/error/manifest.yml

```diff
@@ -1,4 +1,6 @@
 title: Nginx Error Logs
+type: logs
+ingest_pipline: pipeline
```

Comment:
would it make sense to have a default name for the main pipeline? it would make this setting optional I guess

Reply:
yes, we could definitively do that, for example call it `default.json`. And then on package building, it checks if it is there. +1 on this. Could you file an issue for it?

Reply:
https://github.com/elastic/package-registry/issues/183